layers/meta-resin-allwinner/recipes-bsp/brcm-patchram-plus/brcm-patchram-plus.bb

```diff
@@ -0,0 +1,15 @@
+DESCRIPTION = ""
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=86d3f3a95c324c9479bd8986968f4327"
+
+SRC_URI = "git://github.com/balena-os/brcm_patchram_plus"
```

Comment:
We should switch to this one: https://github.com/balena-os/meta-artik710/blob/e8383b7566ebbff9bdf996b5634613469ba845ec/recipes-bsp/broadcom-bluetooth/brcm-patchram-plus_git.bb#L10